src/app.js

```diff
@@ -0,0 +1,87 @@
+const list = document.querySelector('.task-content');
+
+let LIST = [];
+
+const addToDo = (toDo, id, done, trash) => {
+  if (trash) {
+    return;
+  }
+
```

Comment:
- [ ] It would be better if you called the property `id` as `index`. kindly fix this minor issue. 🛠️ 😄 

- [ ] Please make sure that the value of the index is set to the value of the new array length. in your case is the value of the id. 

![Capture](https://user-images.githubusercontent.com/64173189/149567332-557f9593-0506-4052-bed9-5b0802267697.JPG)


![Capture](https://user-images.githubusercontent.com/64173189/149567272-bcc5416b-d0f0-4168-966f-d20c6ff0abda.JPG)

Reply:
Thank you for your professional review, all suggestions implemented.